edgelet/aziot-edged/src/image_gc.rs

```diff
@@ -40,7 +41,15 @@ async fn garbage_collector(
     // track images associated with extant containers
 
     // first get list of containers on the device, running or otherwise
-    let running_modules = ModuleRuntime::list_with_details(runtime).await.unwrap();
+    let running_modules = match ModuleRuntime::list_with_details(runtime).await {
+        Ok(modules) => modules,
+        Err(err) => {
+            return Err(EdgedError::new(format!(
+                "Error in image auto-pruning task: {}; skipping run",
```

Comment:
nit: semicolon seems weird, we can clean up later though

Reply:
Recommend giving more descriptive error, specifically about how we can't get the running containers